Firebase/Storage/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
+# v3.0.0
+- [removed] Removed `downloadURLs` property on `StorageMetadata`. Use `StorageReference.downloadURL(completion:)` to obtain a current download URL.
+- [changed] The `maxOperationRetryTime` timeout now applies to calls to `StorageReference.getMetadata(completion:)`.
```

Comment:
instead of `maxDownloadRetryTime`

Reply:
Can we also change `UpdateMetadata` to use `maxOperationRetryTime` instead of `maxUploadRetryTime`? It seems like that should also obey the general operation one.  https://github.com/firebase/firebase-ios-sdk/blob/b6613bdba73eaf542373ea11a9afeb04cf3b4743/Firebase/Storage/FIRStorageUpdateMetadataTask.m#L51

Reply:
Done (to both).